<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasketWindow</class>
 <widget class="QWidget" name="BasketWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Your basket</string>
  </property>
  <widget class="QLabel" name="basketLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Your basket:</string>
   </property>
  </widget>
  <widget class="QListView" name="basketListView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>731</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="totalSumLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total sum:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalSumEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="basketBackButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>400</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
